### DISCUSSION TITLE
questions

### DISCUSSION THREAD

**Discussion by lihuan2023** [ordinals/ord#1549] [Q&A]
The Essence of NFT on BTC is because currently btc does not have fungibility, and the smallest unit of btc, 1 satoshi, is marked as a specific serial number of NFT.
So if one day the BTC network is successfully upgraded to support fungibility, then the question arises, are these NFTs still NFTs?

**Comment by tyjvazum** [comment]
Bitcoin is fungible today, at a degree that's at least comparable to other large assets. It's certainly more fungible than fiat, especially digital fiat, and I don't think many people would claim that fiat doesn't meet the definition of fungibility.

What you seem to be describing would be more like "perfect privacy", which would be very unlikely to ever be added to the base layer of Bitcoin. No asset in the world currently has anywhere close to that level of privacy, with the movement of gold, oil, and wheat, just as examples, routinely being tracked, even when the owners of the asset don't want it tracked.

So, I don't think the issue is a particularly worrying concern, and it reduces to "what if the fundamental structure of Bitcoin changes", which can't really be addressed prior to that happening. If it did happen, people interested in preserving the value of their NFTs would be incentivized to come up with something they view as a solution.
